CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 * **Backwards-incompatible** squashed migrations:
   If you are currently on a release < 1.2.0, you will need to first install 1.2.x then `manage.py migrate` before
   upgrading to >= 1.3.0.
+* Added locale folder to project.
```

Comment:
no need to give this much detail as the locale folder is there because of the next line.